js/game.js

```diff
@@ -11,23 +11,27 @@ var Question = function (answer, question, message) {
  this.difference = 0;
```

Comment:
I would recommend rearranging some of the property assignments for readability. For example; you might do something like this... 
```
var Question = function (answer, question, message) {
  this.answer = answer;
  this.question = question;
  this.message = message;
  this.guess = 0;
  // etc...
}